examples/dot_product/src/main.cpp

```diff
@@ -0,0 +1,40 @@
+//==============================================================
+// Copyright © 2017-2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+// oneDPL headers should be included before standard headers
+#include <oneapi/dpl/algorithm>
+#include <oneapi/dpl/numeric>
+#include <oneapi/dpl/execution>
+#include <vector>
+#include <random>
+#include <iostream>
+
+double random_number_generator() {
+    // usage of thread local random engines allows running the generator in concurrent mode
+    thread_local static std::default_random_engine rd;
```

Comment:
Just a question: does it make sense to use oneapi::dpl:: RNG functionality in such examples?

Reply:
The thing this example is for host execution (uses TBB, don't use SYCL underneath). Do you think it's better to use oneDPL RNG with TBB backend as well?

Reply:
In this case no need to use oneDPL RNGs here. Thank you!